packages/cashier/src/Sass/cashier.scss

```
@@ -1396,3 +1396,55 @@ div.payment-agent-transfer {
     }
 }
 /* stylelint-enable plugin/selector-bem-pattern */
+
+.cashier-default {
```

only a suggestion
we defined cashier class on line 18. I think you can use &-default and also it's better if you use BEM structure 
```suggestion
.&-default {
```